rpctest/rpc_harness.go

```diff
@@ -317,6 +326,15 @@ func (h *Harness) CreateTransaction(targetOutputs []*wire.TxOut,
 	return h.wallet.CreateTransaction(targetOutputs, feeRate)
 }
 
+// FreeLockedOutputs unlocks any outputs which were previously marked as
+// unspendabe due to being selected to fund a transaction via the
+// CreateTransaction method.
+//
+// This function is safe for concurrent access.
+func (h *Harness) FreeLockedOutputs(inputs []*wire.TxIn) {
```

Comment:
Done.

On Fri, Aug 19, 2016 at 4:35 PM Dave Collins <notifications@github.com>
wrote:

> In rpctest/rpc_harness.go
> <https://github.com/btcsuite/btcd/pull/371#discussion_r75554879>:
>
> > @@ -317,6 +326,15 @@ func (h *Harness) CreateTransaction(targetOutputs []*wire.TxOut,
> >  	return h.wallet.CreateTransaction(targetOutputs, feeRate)
> >  }
> >
> > +// FreeLockedOutputs unlocks any outputs which were previously marked as
> > +// unspendabe due to being selected to fund a transaction via the
> > +// CreateTransaction method.
> > +//
> > +// This function is safe for concurrent access.
> > +func (h *Harness) FreeLockedOutputs(inputs []*wire.TxIn) {
>
> Same deal here if the name of the wallet method is changed to
> UnlockOutputs. And the docs above then could say are locked in order to
> avoid potential ... MUST be unlocked via a call to UnlockOutputs ...
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/btcsuite/btcd/pull/371/files/2925fbd6bf40359007399e1bc5295f12f66cf9aa..2a3c3a04f6351c7e2a7e1d719fe77eb5a201ef6f#r75554879>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA87LoX9gS2TBGC8EVXoNCz1Ji-wVF0eks5qhiGngaJpZM4D7zxE>
> .
>